argocd_application_sets/managing-app-sets-in-non-control-plane-namespaces.adoc

```diff
@@ -11,19 +11,19 @@ include::snippets/technology-preview.adoc[]
 
 By using application sets, you can automate and manage the deployments of multiple Argo CD applications declaratively from a single mono-repository to many clusters at once with greater flexibility.
 
-With {gitops-title} 1.12 and later, you can manage the `ApplicationSet` resources in non-control plane namespaces by explicitly enabling and configuring the `ArgoCD` and `ApplicationSet` custom resources (CRs) as per your requirements. This functionality is particularly useful in multitenancy environments when you want to manage deployments of Argo CD applications for your isolated teams.
+With {gitops-title} 1.12 and later, as a cluster administrator, you can create and manage the `ApplicationSet` resources in non-control plane namespaces declaratively, other than the `openshift-gitops` control plane namespace,  by explicitly enabling and configuring the `ArgoCD` and `ApplicationSet` custom resources (CRs) as per your requirements. This functionality is particularly useful in multitenancy environments when you want to manage deployments of Argo CD applications for your isolated teams. This functionality is called the _ApplicationSet in any namespace_ feature in the Argo CD open source project.
 
 [NOTE]
 ====
 The generated Argo CD applications can create resources in any non-control plane namespace. However, the application itself will be in the same namespace as the application set resources.
 ====
 
-[id="prerequisites_enabling-app-sets-in-any-non-control-plane-namespaces"]
+[id="prerequisites_managing-app-sets-in-non-control-plane-namespaces_{context}"]
 == Prerequisites
 
-* You have a xref:../declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc#using-argo-cd-instance-to-manage-cluster-scoped-resources_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations[cluster-scoped Argo CD instance].
+* You have a user-defined xref:../declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc#using-argo-cd-instance-to-manage-cluster-scoped-resources_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations[cluster-scoped Argo CD instance] in your defined namespace. For example, `spring-petclinic` namespace.
 
-* You have link:https://argocd-operator.readthedocs.io/en/latest/usage/apps-in-any-namespace[explicitly enabled and configured] the target namespaces in the `ArgoCD` CR to manage application resources in non-control plane namespaces.
+* You have xref:../argocd_applications/managing-apps-in-non-control-plane-namespaces.adoc#managing-apps-in-non-control-plane-namespaces[explicitly enabled and configured] the target namespaces in the `ArgoCD` CR to manage application resources in non-control plane namespaces.
```

Comment:
the context used in this assembly is: :context: managing-app-sets-in-non-control-plane-namespaces